maintainers/maintainer-list.nix

```diff
@@ -3039,6 +3039,16 @@
     github = "lnl7";
     name = "Daiderd Jordan";
   };
+  l0b0 = {
```

Comment:
Can the addition to `maintainer-list.nix` be in a separate commit?

Reply:
Is this OK?

Reply:
It would be better if the commit was named `Add l0b0 to maintainers.nix` but that is the right structure

Reply:
Meet you halfway? :)

Reply:
That's great, thanks

Reply:
generally I see "maintainers: add l0b0"

https://github.com/NixOS/nixpkgs/commits/6f1445ce9eee3ee3e7efe5bedd5c2a4718671c30/maintainers/maintainer-list.nix

Reply:
that looks familiar and is what I was trying to remember for my first suggestion I think

Reply:
And done.